src/scripts/main.js

```diff
@@ -1,3 +1,74 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const table = document.querySelector('table');
+const body = document.querySelector('body');
+const rows = table.rows;
+let rowCount = 4;
+let columnCount = 4;
+const addRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const addColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+
+addRowButton.addEventListener('click', () => {
+  const rowTr = tbody.children[0].cloneNode(true);
+
+  tbody.append(rowTr);
+
+  rowCount++;
+});
+
+removeRowButton.addEventListener('click', () => {
+  tbody.deleteRow(-1);
```

Comment:
Consider adding a condition to check if `rowCount` is greater than the minimum allowed value (e.g., 2) before deleting a row. This will prevent errors when trying to delete rows below the minimum limit.